public/locales/en-US.json

```diff
@@ -428,6 +434,9 @@
     "title_rhel-for-x86-ha": "$t(curiosity-view.title_RHEL)",
     "subtitle_rhel-for-x86-ha": "$t(curiosity-view.title_RHEL)",
     "description_rhel-for-x86-ha": "$t(curiosity-view.description_RHEL)",
+    "title_rhel-for-x86-rs": "$t(curiosity-view.title_RHEL)",
+    "subtitle_rhel-for-x86-rs": "$t(curiosity-view.title_RHEL)",
```

Comment:
discovered after the last update... this line `438` should be
```
"subtitle_rhel-for-x86-rs": "$t(curiosity-view.subtitle_RHEL)",
```

Reply:
well spotted! do you want me to change the others too ?

Reply:
okis, PR updated.